cmd/producer/main.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"github.com/ONSdigital/dp-hierarchy-builder/config"
+	"github.com/ONSdigital/dp-import/events"
+	"os"
+	"time"
+
+	kafka "github.com/ONSdigital/dp-kafka/v2"
+	"github.com/ONSdigital/log.go/log"
+)
+
+var instanceID = flag.String("instance-id", "58004716-a2d4-4dd1-a6c3-6accab30ad2a", "")
+var codeListID = flag.String("code-list-id", "cpih1dim1aggid", "")
+var dimensionName = flag.String("dimension-name", "aggregate", "")
+
+func main() {
+	ctx := context.Background()
+	flag.Parse()
+
+	var brokers []string
+	brokers = append(brokers, "localhost:9092")
```

Comment:
Can the `broker` by pulled from the configs instead of inlined:

`	cfg := &Config{
		KafkaAddr:                  []string{"localhost:9092"},`

Reply:
A good point Mike - I ran into this problem when trying to use the producer in develop. I have made this change in another PR: https://github.com/ONSdigital/dp-hierarchy-builder/pull/103

Reply:
👍